### DISCUSSION TITLE
Synchronisation of two custom Mediastreamtracks

### DISCUSSION THREAD

**Discussion by Alpe6825** [aiortc/aiortc#908] [Q&A]
I have a server that generates audio+video content, which should send to a client via webrtc. 

I used the `VideoStreamTrack` and `AudioStreamTrack` as wrapper. Unfortunately at the client both streams are not in sync.
Any idea how i could debug and fix this?

Further I have some questions about `VideoStreamTrack` and `AudioStreamTrack`. 
Why are the `AUDIO_PTIME = 0.020`  and `VIDEO_CLOCK_RATE = 90000`?

**Comment by abmohamedart** [comment]
@Alpe6825 did you find the solution for that?
